cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -1344,7 +1437,13 @@ func TestWatchListClientFlagChange(t *testing.T) {
 
 	args := []string{fmt.Sprintf("--feature-gates=%v=true", clientgofeaturegate.WatchListClient)}
 	if err := fs.Parse(args); err != nil {
-		t.Fatal(err)
+		t.Fatal(fmt.Errorf("FlatSet.Parse failed with %w", err))
+	}
+
+	// this is needed to Apply parsed flags to GlobalRegistry, so FeatureGates() can reference from the registry.
```

Comment:
nit: `clientgofeaturegate.FeatureGates()` is not in the registry. 
Can you change it to 
`// this is needed to Apply parsed flags to GlobalRegistry, so the DefaultFeatureGate values can be set from the flag`